Automattic/jetpack
# Record meter: format the numbers in each bar entry.

#### Changes proposed in this Pull Request:

Related: #24810

Let's use formatted numbers whenever possible, it looks better with bigger numbers.

**Before**

<img width="853" alt="Screen Shot 2022-06-22 at 15 15 46" src="https://user-images.githubusercontent.com/426388/175040769-f8827282-8067-4234-b558-d2f93730ffd1.png">

**After**

<img width="833" alt="Screen Shot 2022-06-22 at 15 24 45" src="https://user-images.githubusercontent.com/426388/175040712-76263b44-ac96-4ab0-8b33-59671f8f83ec.png">

#### Other information:


#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start with a site with a Jetpack Search plan, and with more than 1,000 posts.
* Go to `yoursite.com/wp-admin/admin.php?page=jetpack-search&features=record-meter`
* Notice that the post count is now formatted.
